src/providers/ethereumProviderWrapper.ts

```diff
@@ -67,6 +68,17 @@ export class EthereumProviderWrapper {
       return ERC725_VERSION.ERC725;
     }
 
+    const isErc725Yv200 = await this.supportsInterface(
+      address,
+      INTERFACE_IDS.ERC725Y_v200,
+    );
+
+    if (isErc725Yv200) {
+      return ERC725_VERSION.ERC725;
+    }
+
+    // v2.0.0 and v6.0.0 have the same function signatures for getData, only versions before v2.0.0 requires a different call
```

Comment:
I think these versions are wrong. Current smart contract version is v0.6.0

Reply:
ah indeed my bad, it is v0.2.0 instead of v2.0.0 - thanks will update